newserving/Dockerfile

```diff
@@ -29,5 +29,6 @@ FROM alpine
 COPY --from=0 ./app/main ./main
 COPY --from=0 ./app/provider/scripts/offline_store_spark_runner.py /app/provider/scripts/offline_store_spark_runner.py
 
+ENV SERVING_PORT "0.0.0.0:8080"
```

Comment:
would this make address 0.0.0.0:0.0.0.0:8080 then?

Reply:
fixed